specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
It is critical for someone to confirm that the CVR data we're auditing has the same outcome as the reported outcome, and that the margins reflected in it match the margins we're using for the sample size calculations. Otherwise, they could send us CVRs that match the ballots but not the announced results, or set the margin at 50%, or all kinds of things. The natural thing is for the tool to do that, if nothing else to protect against CVR upload issues, bugs in the code etc. That is a standard part of an RLA, but it is in some sense separable from the data gathering that the tool is doing.
We could do the calculations as a check during the importing of the data, without storing the CVRs in that sort of database table. I see that as easier than parsing the Dominion tabulation results file. We already have `rla_tabulation_outcome` for storing the tabulation outcomes, which should also have margin information.